src/style.css

```diff
@@ -0,0 +1,138 @@
+/* #region body */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12;
+  color: #000;
+}
+/* #endregion */
+/* #region card */
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 29px 14px 14px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 0.5px;
+}
+/* #endregion */
+/* #region image */
+.card__image {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin-bottom: 45px;
+}
+
+.iMac-image {
+  width: 160px;
+  height: 134px;
+}
+/* #endregion */
+/* #region description */
+.description {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 15px;
```

Comment:
don't give margins to BEM-block. Better to remove this block and make it an element of another block, as it doesn't look like an independent logical block that can be reused.